aerogear/mobile-services-installer
# Initial version of installer

### Overview
* moved `minishift` and `oc-cluster-up` scripts from [mdc repo](https://github.com/aerogear/mobile-developer-console/tree/master/scripts)
* created role for adding mobile crd & mobileclient admin role
* created role for configuration of ansible broker

### Comments
* `yq` YAML processor is not used anymore, YAML patching is done purely with ansible

### Verification steps
1. Checkout this PR
1. Run `./scripts/minishift.sh`
1. Check broker config (`oc get configmap broker-config -n openshift-automation-service-broker -o jsonpath='{.data.broker-config}'`) and verify that configuration from [this](https://github.com/aerogear/mobile-services-installer/pull/1/files#diff-0765906c27aa8bc794fb8e05c43edf59R3) & [this file](https://github.com/aerogear/mobile-services-installer/pull/1/files#diff-300560c026bbaeab7409153ca8b67f55R3) was applied
1. Run the playbook for installation of mobile services (`ansible-playbook install-mobile-services.yml`) and verify that the configmap was not changed when no changes were made to local config files (last playbook task should be skipped):
```
TASK [ansible-service-broker-setup : Patch Broker Configmap] **********************************************************************************
skipping: [localhost]
```
